### ISSUE TITLE
Inconsistent execution bug

### ISSUE THREAD

**Issue by akaleeroy** [ahkscript/SublimeAutoHotkey#52]
### Steps to reproduce

```autohotkey
output := ComObjCreate("WScript.Shell").Exec("powercfg /requests").StdOut.ReadAll()  
MsgBox % output
```

Run from Sublime Text it will show an incorrect output, different from the output of running `powercfg /requests` from a command prompt. Run by clicking on the .ahk file it will show the correct output.

Also described in [ComObjCreate("WScript.Shell").Exec("powercfg /requests") returns different output than actual command run in command prompt : AutoHotkey](https://www.reddit.com/r/AutoHotkey/comments/bwlqei/comobjcreatewscriptshellexecpowercfg_requests/)

### Fix

Using AutoHotkeyU32.exe / AutoHotkeyA32.exe on x64 systems runs into this bug.  
[Changing to a 64-bit executable](https://github.com/ahkscript/SublimeAutoHotkey/blob/d21f1bbb69b88ea07262df1c3936374e4a411eff/AutoHotkey.sublime-settings#L7) fixes it.

**AutoHotkey.sublime-settings**

```json
{
  "AutoHotKeyExePath":
  {
    "default": "C:\\Program Files\\AutoHotkey\\AutoHotkey.exe"
  }
}
```

**Comment by robertcollier4** [comment]
Fixed with Commit 8cc1631 - "ahkrun and ahkcompile allow targeting multiple ahk versions"

**Comment by akaleeroy** [comment]
In my opinion fixing the bug would be to automatically pick the correct executable based on system architecture.
